src/valueObjects/AssetProxy.js

```diff
@@ -8,14 +8,16 @@ export const setStore = (storeObj) => {
   store = storeObj;
 };
 
-export default function AssetProxy(value, fileObj, uploaded = false) {
+export default function AssetProxy(value, fileObj, uploaded = false, field = null) {
   const config = store.getState().config;
+  const media_folder = (field && field.has('media_folder')) ? field.get('media_folder') : config.get('media_folder');
+  const public_folder = (field && field.has('public_folder')) ? field.get('public_folder') : '/' + media_folder;
```

Comment:
`public_folder` now falls back to `'/' + media_folder` instead of `config.get('public_folder')`?